<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DanmuDisplay</class>
 <widget class="QWidget" name="DanmuDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DanmuDisplay</string>
  </property>
  <property name="windowOpacity">
   <double>0.700000000000000</double>
  </property>
  <widget class="QListWidget" name="list_danmuDisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>481</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_debug">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_liveRoomInfo">
    <item>
     <widget class="QLabel" name="prompt_followersCount">
      <property name="text">
       <string>关注人数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_followersCount">
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="prompt_viewersCount">
      <property name="text">
       <string>直播人气</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_viewersCount">
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
